src/collectors/AbstractCollector.php

```diff
@@ -162,7 +189,7 @@ protected function renderConfig($rows, $format)
         return $res;
     }
 
-    protected function buildLastTimeJoin()
+    protected function buildLastTimeJoin() : ?string
```

Comment:
```suggestion
    protected function buildLastTimeJoin() : string
```
тут тоже точно строка